lib/views/components/base_view.dart

```diff
@@ -17,22 +19,36 @@ class BaseView extends StatelessWidget {
     this.showLeading = true,
     this.customAppBar,
     this.bottomNavigationBar = const CustomBottomNavBar(),
+    this.settingsHamburgerMenu = const SettingsScreen(),
   });
 
   @override
   Widget build(BuildContext context) {
+    final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
+    bool isTablet = MediaQuery.of(context).size.width >= 600 ? true : false; //const BoxConstraints().maxWidth >= 600 ? true : false;
+    debugPrint("Width: ${MediaQuery.of(context).size.width}");
```

Comment:
remove this print statement

Reply:
Yes that is intended, it checks for screed width rather than device type - like you I thought it looked better, but I can change it too?